pkg/service/replay/replay.go

```diff
@@ -118,16 +125,47 @@ func (r *Replayer) Start(ctx context.Context) error {
 		return fmt.Errorf(stopReason)
 	}
 
+	if !r.config.Test.SkipCoverage && r.config.Test.Language == "typescript" {
```

Comment:
why is this done twice..?

Reply:
You can add clean, append and testset envs irrespective to languages.

Reply:
this is because first clean must be set to true to clean up previous coverage data and then after first test-set has completed execution clean would be set to false

Reply:
A quick solution would be to not append --clean=${CLEAN} in cli/provider/cmd.go and append it in replay.go when we are setting clean env to false(like --clean=false, no need to set env also), but it won't work because command gets saved to app struct through NewApp func invocation. should I implement an updateCmd function in app.go to clean up this code from replay.go ?

Reply:
No what I am telling is this condition `r.config.Test.Language == "typescript"` is not required. You can just do it by default without condition, even the test-set name env.

Reply:
done!